src/encoded/static/scss/encoded/modules/_cart.scss

```diff
@@ -1156,3 +1189,67 @@ $file-type-colors: (
         fill: #000;
     }
 }
+
+// Cart-tools buttons that stay side by side even on mobile.
+.cart-tools-extras {
+    display: flex;
+    align-content: stretch;
+    margin-bottom: 5px;
+
+    @media screen and (min-width: $screen-md-min) {
+        margin-right: 5px;
+        margin-bottom: 0;
+    }
+
+    .cart-tools-extras__button {
+        margin-right: 5px;
+        flex-grow: 1;
+
+        &:last-child {
+            margin-right: 0;
+        }
+
+        button {
+            width: 100%;
+        }
+    }
+}
+
+.cart-dataset-selector {
+    width: 100%;
+    margin-bottom: 5px;
```

Comment:
I would add a little padding here on the right and left.

Reply:
Added 5px left and right.